nessie/models/account.py

```diff
@@ -0,0 +1,28 @@
+class Account:
+
+    TYPES = {"Checking", "Credit Card", "Savings"}
+
+    def __init__(self, _id, type, nickname, rewards, balance, customer_id, account_number=None):
+        self._id = _id
+        self.type = type
+        self.nickname = nickname
+        self.rewards = rewards
+        self.balance = balance
+        self.account_number = account_number
+        self.customer_id = customer_id
+
+    @staticmethod
+    def typeIsValid(accountType):
+        return accountType in Account.TYPES
+
```

Comment:
I've been adding `toDict` methods in case users want the fields in a dictionary form - might be a good thing to add

Reply:
not sure if you commented on the correct place? this was anthony's type check

Reply:
yeah I was just commenting in this file generally, wasn't sure where to put it. I thought a `toDict` method for the Account class might be nice for users and for the sake of consistency since other classes have it